oauth2-authorization-server/src/test/java/org/springframework/security/oauth2/server/authorization/client/TestRegisteredClients.java

```diff
@@ -53,6 +53,18 @@
 				.scope("scope2");
 	}
 
+	public static RegisteredClient.Builder registeredClient3() {
```

Comment:
Let's remove this and reuse an existing `RegisteredClient` template. You can easily adjust the configuration via the returned `Builder`.